### DISCUSSION TITLE
A question about how to build my own iso ?

### DISCUSSION THREAD

**Discussion by souhaiebtar** [SpiralLinux/SpiralLinux-project#415] [General]
Hello,

thank you for your great work, i really like the btrfs and snapshot integration in the grub, i can easily say that it's exactly what i wanted,  i want  to use your github repo to create my own iso (basically removing some stuff and adding some others)

**Comment by geckolinux** [comment]
Hi there @souhaiebtar , I'm glad to hear that SpiralLinux is working well for your needs.

Here is a fairly detailed description about how I build the SpiralLinux ISOs:
-  https://github.com/orgs/SpiralLinux/discussions/321#discussioncomment-7235501
- https://github.com/orgs/SpiralLinux/discussions/228#discussioncomment-4809266

**Reply by souhaiebtar** [reply]
@geckolinux thank you very much for the help
